substrate/frame/babe/src/benchmarking.rs

```diff
@@ -53,22 +55,21 @@ benchmarks! {
 			124, 11, 167, 227, 103, 88, 78, 23, 228, 33, 96, 41, 207, 183, 227, 189, 114, 70, 254,
 			30, 128, 243, 233, 83, 214, 45, 74, 182, 120, 119, 64, 243, 219, 119, 63, 240, 205,
 			123, 231, 82, 205, 174, 143, 70, 2, 86, 182, 20, 16, 141, 145, 91, 116, 195, 58, 223,
-			175, 145, 255, 7, 121, 133
+			175, 145, 255, 7, 121, 133,
 		];
 
 		let equivocation_proof1: sp_consensus_babe::EquivocationProof<Header> =
 			Decode::decode(&mut &EQUIVOCATION_PROOF_BLOB[..]).unwrap();
 
 		let equivocation_proof2 = equivocation_proof1.clone();
-	}: {
-		sp_consensus_babe::check_equivocation_proof::<Header>(equivocation_proof1);
-	} verify {
+
+		#[extrinsic_call]
+		_(equivocation_proof1);
```

Comment:
`check_equivocation_proof` is not a Runtime call, you need `#[block]` here

Reply:
thanks for point this out, fixed. please double check the benchmarking.